Patches/Midworld Expanded Flak Armor/ThingDefs_Misc/ThingDef_Apparel.xml

```diff
@@ -0,0 +1,218 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Patch>
+  <Operation Class="PatchOperationSequence">
+	<operations>
+	  <li Class="PatchOperationFindMod">
+			
+		<mods><li>Midworld Expanded: Flak Armor</li></mods>
+			
+		<match Class="PatchOperationSequence">
+		<operations>
+		
+			<!--Bulk/Mass-->
+			
+			<li Class="PatchOperationReplace">
+				<xpath>Defs/ThingDef[defName="Apparel_FlakGloves" or
+				defName="Apparel_FlakBoots" or
+				defName="Apparel_FlakMask"
+				]/statBases/Mass</xpath>
+				<value>
+					<Mass>0.50</Mass>
+					<Bulk>3</Bulk>
+					<WornBulk>2</WornBulk>
+				</value>
+			</li>
+			
+			<li Class="PatchOperationReplace">
+				<xpath>Defs/ThingDef[
+				defName="FlakShirt"
+				]/statBases/Mass</xpath>
+				<value>
+					<Mass>1.2</Mass>
+					<Bulk>5</Bulk>
+					<WornBulk>3</WornBulk>
+				</value>
+			</li>
+			
+			<li Class="PatchOperationReplace">
+				<xpath>Defs/ThingDef[
+				defName="LightFlakVest"
+				]/statBases/Mass</xpath>
+				<value>
+					<Mass>6.25</Mass>
+					<Bulk>5</Bulk>
+					<WornBulk>3</WornBulk>
+				</value>
+			</li>
+			
+			<li Class="PatchOperationReplace">
+				<xpath>Defs/ThingDef[
+				defName="LightFlakVest"
+				]/statBases/MaxHitPoints</xpath>
+				<value>
+					<MaxHitPoints>100</MaxHitPoints>
+				</value>
+			</li>
+			
+			<li Class="PatchOperationReplace">
+				<xpath>Defs/ThingDef[
+				defName="HeavyFlakSuit"
+				]/statBases/Mass</xpath>
+				<value>
+					<Mass>17.5</Mass>
+					<Bulk>60</Bulk>
+					<WornBulk>14</WornBulk>
+				</value>
+			</li>
+			
+			<!--Armor Values-->
+			
+			<li Class="PatchOperationReplace">
+				<xpath>Defs/ThingDef[defName="Apparel_FlakGloves" or
```

Comment:
Since there's no penalty to dexterity (and I'm not suggesting we add one), I'd decrease the armor of the gloves slightly, as they'd have to be fairly thin. I'd move it down with the `FlakShirt` to have 1 thickness and 0.5 sharp.